xml/mysql_banners.xml

```diff
@@ -132,6 +138,8 @@
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="2" name="os.version"/>
+    <param pos="0" name="os.certainty" value="0.80"/>
```

Comment:
More certain or more complete?  I don't want to derail this review, so for now I'd say don't make any changes to the existing certainty values for mysql in recog.  Eventually I'd like to see the certainty values rethought and I imagine what will come of that is that the certainty values will be removed from recog.  Obviously that is a big change with a big impact on things that integrate recog so it will take some time and discussions.